flow_action_components/ExecuteSOQLQuery/force-app/main/default/classes/ExecuteSOQL.cls

```diff
@@ -7,14 +8,15 @@ public with sharing class ExecuteSOQL {
         for(Requests curRequest : requestList) {
             String soqlQuery = curRequest.soqlQuery;
             soqlQuery = replaceWithFormattedValues(soqlQuery);
+            System.debug('Logs: query: ' + soqlQuery);
             results.sObjects = Database.query(soqlQuery);
             responseWrapper.add(results);
         }
         return responseWrapper;
     }
     public static String replaceWithFormattedValues(String soqlQuery) {
         String endingClause = '';
-        soqlQuery = soqlQuery.toLowerCase().replaceAll('\r\n|\n|\r|\t',' ');
+        soqlQuery = soqlQuery.replaceAll('\r\n|\n|\r|\t',' ');
```

Comment:
Bonus fix: by lower casing the query string rather than doing a case insensitive regex search, queries like this break because Id is a case-sensitive field: 
```
Select Id from Contact where AccountId = 'x'
```